src/pages/GetStarted/GetStartedPlaces.vue

```diff
@@ -505,6 +562,11 @@ $box-shadow-hover: 0 rem-calc(2) rem-calc(10) 0 rgba(0, 0, 0, 0.35);
 		}
 	}
 
+	&__circle {
```

Comment:
Possibly fixes that issue we're seeing on dev, where when clicking next, the country names scoot left